earth_enterprise/BUILD_RPMS.md

```diff
@@ -0,0 +1,23 @@
+# Installing via Package Managers
+This document describes how to build and install Open GEE packages (i.e. RPMs, and later, DEBs).  Packages must be built on the same platform that they are installed on to avoid library errors.  This document assumes you have development system of the target platform set up and ready to build Open GEE.
+
+Currently only building RPM packages are supported.  To begin creating opengee RPMs, you first start with a normal Open GEE build environment following the existing [build instructions](https://github.com/google/earthenterprise/wiki/Build-Instructions), such as those already found in **BUILD.md**.  You should test this environment building a **stage_install** to verify everything builds correctly before attempting to create RPMs.  You should **NOT** use the ```install_server.sh``` or ```install_fusion.sh``` scripts.
+
+## Prerequisites
+The RPMs are built using gradle, and gradle requires Java.  On Ubuntu (16.04) one can use ```apt-get install default-jdk rpm```.  The minimum requirement is Java 7 JDK.  You can build RPMs on Ubuntu, and therefore test the build process itself there, but you cannot actually install the resulting RPMs anywhere.  To create RPM's that will install on Centos or RHEL 6, you must build your RPM's on the same platform.
+
+On Centos or RHEL you should be able to do a ```yum install java-1.7.0-openjdk-devel```.  There are no other special prerequesets, as the scons build target pulls down osPackage and dependencies as needed before creating rpms.
+
+
+## Creating RPMs
+A new scons target was added to make RPMs, **package_install**, that can be ran from the *earth_enterprise* subdirectory.  This can be used after **stage_install** to verify the build, as it uses the stage install directory.  This will setup the stage install directory, install gradle if needed, and then uses gradle to generate RPMs.  The RPMs are created in the *rpms/build/distributions* subdirectory.  A typical invocation for building release RPMs might be ```scons -j8 release=1 package_install```.  All other files related to RPM generation are found in the *rpms* sub-directory as well.  Scripts for pre/post install operations are found under the related rpm directory in *rpms* and common templates for package script generation in *rpms/shared*.
```

Comment:
Grammar: `that can be ran` -> `that can be run`.

Reply:
Minor: Capitalize `gradle`.

Reply:
Minor: Capitalize `scons`.

Reply:
Suggestion: Provide a series of commands in a stand-alone block that build Open GEE, and produce the RPMs (and, maybe, even one for the Debs).